packages/react-data-grid/src/focusableComponentWrapper.js

```diff
@@ -8,12 +8,17 @@ const focusableComponentWrapper = WrappedComponent => {
       constructor() {
         super();
         this.checkFocus = this.checkFocus.bind(this);
+        this.state = { isScrolling: false };
       }
 
       shouldComponentUpdate(nextProps) {
         return WrappedComponent.isSelected(this.props) !== WrappedComponent.isSelected(nextProps);
       }
 
+      componentWillReceiveProps() {
```

Comment:
We need at least to do a shallow compare on the cellMetaData of the current props and the nextProps otherwise it will kill the grid performance to set the state every time we receive new props.